src/Groups/GAPGroups.jl

```diff
@@ -0,0 +1,617 @@
+
+"""
+    isconjugate(G::Group, x::GAPGroupElem, y::GAPGroupElem)
+if `x`,`y` are conjugate in `G`, return 
+```
+true, z
+```
+where `x^z=y`; otherwise, return
+```
+false, nothing
+```
+"""
+function isconjugate(G::Group, x::GAPGroupElem, y::GAPGroupElem)
+   if GAP.Globals.IsConjugate(G.X, x.X, y.X)
+      return true, group_element(G,GAP.Globals.RepresentativeAction(G.X, x.X, y.X))
+   else
+      return false, nothing
+   end
+end
+# END elements conjugation 
+
+# START subgroups conjugation
+"""
+    conjugacy_class(G::T, H::T) where T<:Group -> GroupConjClass
+return the subgroup conjugacy class `cc` of `H` in `G`, where `H` = `representative`(`cc`).
+"""
+function conjugacy_class(G::T, g::T) where T<:Group
+   return _conjugacy_class(G, g, GAP.Globals.ConjugacyClassSubgroups(G.X,g.X))
+end
+
+function Base.rand(C::GroupConjClass{S,T}) where S where T<:Group
+   return T(GAP.Globals.Random(C.CC))
+end
+
+function elements(C::GroupConjClass{S, T}) where S where T<:Group
+   L=GAP.Globals.AsList(C.CC)
+   l = Vector{T}(undef, length(L))
+   for i in 1:length(l)
+      l[i] = _as_subgroup(L[i],C.X)[1]
+   end
+   return l
+end
+
+"""
+    conjugacy_classes_subgroups(G::Group)
+returns the array of all conjugacy classes of subgroups of G.
+"""
+function conjugacy_classes_subgroups(G::Group)
+   L=GAP.gap_to_julia(GAP.Globals.ConjugacyClassesSubgroups(G.X))
+   return GroupConjClass{typeof(G), typeof(G)}[ _conjugacy_class(G,typeof(G)(GAP.Globals.Representative(cc)),cc) for cc in L]
+end
+
+"""
+    conjugacy_classes_maximal_subgroups(G::Group)
+returns the array of all conjugacy classes of maximal subgroups of G.
+"""
+function conjugacy_classes_maximal_subgroups(G::Group)
+  L = GAP.gap_to_julia(GAP.Globals.ConjugacyClassesMaximalSubgroups(G.X))
+   return GroupConjClass{typeof(G), typeof(G)}[ _conjugacy_class(G,typeof(G)(GAP.Globals.Representative(cc)),cc) for cc in L]
+end
+
+Base.:^(H::Group, y::GAPGroupElem) = typeof(H)(H.X ^ y.X)
+
+"""
+    isconjugate(G::Group, H::Group, K::Group)
+if `H`,`K` are conjugate subgroups in `G`, return 
+```
+true, z
+```
+where `H^z=K`; otherwise, return
+```
+false, nothing
+```
+"""
+function isconjugate(G::Group, H::Group, K::Group)
+   if GAP.Globals.IsConjugate(G.X, H.X, K.X)
+      return true, group_element(G,GAP.Globals.RepresentativeAction(G.X, H.X, K.X))
+   else
+      return false, nothing
+   end
+end
+
+# END subgroups conjugation
+
+
+################################################################################
+#
+# Normal Structure
+#
+################################################################################
+
+"""
+```
+    normalizer(G::Group, H::Group)
+    normaliser(G::Group, H::Group)
+```
+returns `N,f`, where `N` is the normalizer of `H` in `G` and `f` is the embedding morphism of `N` into `G`.
+"""
+normalizer(G::T, H::T) where T<:Group = _as_subgroup(GAP.Globals.Normalizer(G.X,H.X),G)
+
+"""
+```
```

Comment:
It seems better to not add these triple quotes for the signature, the dosctring system already handles that.